### PR TITLE
Remove minimal.css

### PR DESCRIPTION
**Pull request by aabrook**
It finally runs in supporter!

Looks a bit average but it's there ...
screen shot 2016-11-08 at 5 05 20 pm

### CODE REVIEW

File: source/client.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 import { render } from 'react-dom'
+import images from './images'
```
Comment (bradparker) [inline source/client.js]: Where be?

Comment (aabrook) [inline source/client.js]: https://github.com/everydayhero/nps-collect/blob/master/source/images.js

**Comment by lukebrooker** [comment]
Nice!

…the background is grey?

**Comment by aabrook** [comment]
That's the default background. Other pages look like:
screen shot 2016-11-08 at 5 31 38 pm

**Pull request merged**